sirepo/package_data/template/activait/train.py.jinja

```diff
@@ -18,7 +21,14 @@ history = model.fit(
 
 {% else %}
 
-model.compile(loss='{{ neuralNet_losses }}', optimizer='{{ neuralNet_optimizer }}')
+model.compile(
+{% if "keras" in neuralNet_losses %}
+    loss={{ neuralNet_losses }},
+{% else %}
+    loss='{{ neuralNet_losses }}',
+{% endif %}
```

Comment:
This part is a little awkward. How about have the generated code like:
```
   loss=keras.losses.{{ neuralNet_losses }}({{ lossArguments }}),
```
And then set lossArguments to '' by default in template.activait, and to 'from_logits=True' in the case where you need it. That way you don't need to look for 'keras' in the loss value which is confusing.